.kitchen.yml

```diff
@@ -22,13 +22,17 @@ driver:
   name: vagrant
   customize:
     memory: 1024
+  network:
+    - ["private_network", {ip: "33.33.33.200"}]
```

Comment:
I used it during development so I could get direct access to the machines, forgot to remove it. It doesn't hurt that it's there, but I can remove it, if you'd prefer.